src/main/resources/liquibase/crop_changelog/15_1_0.xml

```diff
@@ -19,4 +19,115 @@
 		<modifyDataType tableName="key_sequence_register" columnName="key_prefix" newDataType="varchar(255)"/>
 	</changeSet>
 
+	<changeSet author="aldrin" id="v15.1.0-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="stock" columnName="project_id"/>
```

Comment:
@darla-leafnode we should add an extra changeset for the FK between stock and project.